app/routes/bestuurseenheid/zitting/besluitenlijst/index.js

```diff
@@ -1,24 +1,33 @@
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
 export default class BestuurseenheidZittingBesluitenlijstIndexRoute extends Route {
-  breadCrumb = { title: 'Besluitenlijst' };
+  @service store;
 
   async model() {
-    let zitting = this.modelFor('bestuurseenheid.zitting');
+    let zitting = await this.modelFor('bestuurseenheid.zitting');
```

Comment:
I don't think the await is needed here? modelFor should be sync since the parent's model is already resolved at this point.